server/routes/api/documents/documents.ts

```diff
@@ -1975,9 +1976,8 @@ router.post(
       paranoid: false,
     });
 
-    await documentPermanentDeleter(documents);
-    await Event.createFromContext(ctx, {
```

Comment:
Removed unnecessary event which was added in the initial implementation.

Reply:
I think the event is useful actually because this is a very destructive action… without it there is no record of who permanently deleted data 🙈